### ISSUE TITLE
Product vision

### ISSUE THREAD

**Issue by tawfeeka** [amosproj/amos2023ss02-open-search-meta-data-hub#19]
## User story
1. As a Team
2. I need a product vision that describes how the product can address and solve the problem or needs of our customers optimally. 
3. So that every team member is supported and inspired on the path towards a solution that delights our customers.

## Acceptance criteria
* This should be well documented in the planning document.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by motrell** [comment]
Have read through it, sounds good!
